ui/src/containers/SampleQueueContainer.js

```diff
@@ -4,56 +4,35 @@ import { connect } from 'react-redux';
 import CurrentTree from '../components/SampleQueue/CurrentTree';
 import TodoTree from '../components/SampleQueue/TodoTree';
 import QueueControl from '../components/SampleQueue/QueueControl';
-import * as queueActions from '../actions/queue'; // eslint-disable-line import/no-namespace
-import * as queueGUIActions from '../actions/queueGUI'; // eslint-disable-line import/no-namespace
-import * as sampleViewActions from '../actions/sampleview'; // eslint-disable-line import/no-namespace
-import * as sampleChangerActions from '../actions/sampleChanger'; // eslint-disable-line import/no-namespace
+import {
+  sendToggleCheckBox,
+  sendPauseQueue,
+  sendUnpauseQueue,
+  sendStopQueue,
+  changeTaskOrderAction,
+  deleteTask,
+  addTask,
+  moveTask,
+  setAutoMountSample,
+  setAutoAddDiffPlan,
+  sendRunSample,
+  sendSetCentringMethod,
+  setEnabledSample,
+} from '../actions/queue';
+import {
+  collapseItem,
+  selectItem,
+  showConfirmCollectDialog,
+  showList,
+} from '../actions/queueGUI';
 import { showTaskForm } from '../actions/taskForm';
 import { Nav } from 'react-bootstrap';
 import { showDialog } from '../actions/general';
 
 import UserMessage from '../components/Notify/UserMessage';
 import loader from '../img/loader.gif';
 import { prepareBeamlineForNewSample } from '../actions/beamline';
-
```

Comment:
Ugly diff because I've moved this down just above the call to `connect`, like in every other file.